src/jmp.zig

```diff
@@ -1,3 +1,8 @@
-pub const jmp = @cImport({
+// FIXME: remove setjmp/longjmp  https://github.com/ziglang/zig/issues/1656#issuecomment-840190292
```

Comment:
i'm aware of this issue, but we don't run into it since it's really mir code that calls it